src/test/java/pokemanager/StoreCommandTest.java

```diff
@@ -0,0 +1,36 @@
+package pokemanager;
+
+import org.junit.Test;
+
+import java.io.*;
+
+import static org.junit.Assert.*;
+
+public class StoreCommandTest {
+
+    private final ByteArrayOutputStream out = new ByteArrayOutputStream();
+    private final PrintStream pw = new PrintStream(out);
+    private final BoxSpy box = new BoxSpy(new ByteArrayInputStream("".getBytes()));
```

Comment:
I would call this variable `boxSpy` just so that there's no confusion that this isn't the real box but just a spy.